tools/run_integration_tests.sh

```diff
@@ -50,14 +50,30 @@ check_required_params() {
   fi
 }
 
-export HDFS_ROOT=file:///tmp
-
 check_required_params
 
 # When tests run, they run in the root of the module directory. Anything
 # relative to our current directory won't work properly
-GCS_TEST_PRIVATE_KEYFILE=$(readlink -f "${GCS_TEST_PRIVATE_KEYFILE}")
+if [[ $(uname -s) == Darwin ]]; then
+  # On MacOS `readlink` does not support `-f` parameter
+  # so we need manually resolve absolute path and symlink after that
+  abs_path() {    
+   echo "$(cd $(dirname "$1"); pwd)/$(basename "$1")"
+  }
+  GCS_TEST_PRIVATE_KEYFILE=$(abs_path "$GCS_TEST_PRIVATE_KEYFILE")
```

Comment:
be consistent and use curly brackets ${GCS_TEST_PRIVATE_KEYFILE}

Reply:
Done.